packages/ts/react-grid/test/autocrud.spec.tsx

```diff
@@ -105,5 +112,64 @@ describe('@hilla/react-grid', () => {
       await nextFrame();
       expect(getBodyCellContent(grid, 1, 1).innerText).to.equal('2');
     });
+    it('shows a confirmation dialog before deleting', async () => {
+      const service = personService();
+      const result = render(<ExperimentalAutoCrud service={service} model={PersonModel} />);
+      const grid: GridElement<Person> = getGrid(result);
+      await nextFrame();
+      await nextFrame();
+      await nextFrame();
+      const cell = getBodyCellContent(grid, 1, 5);
+      (cell.querySelector('vaadin-button') as HTMLElement).click();
+      await nextFrame();
+      await nextFrame();
+      const dialog = cell.querySelector('vaadin-confirm-dialog');
+      const content = (dialog as any).$.dialog.$.overlay.$.content as HTMLElement;
+      const text = content
+        .querySelector('slot')!
+        .assignedNodes()
+        .map((e) => (e as HTMLElement).innerText)
+        .join('');
```

Comment:
This can be extracted into a utility function to prevent repeating it.

Reply:
Sure but it is not repeated anywhere?

Reply:
What about 
```tsx
      const dialog = cell.querySelector('vaadin-confirm-dialog');
      const content = (dialog as any).$.dialog.$.overlay;
      const confirmButton = content?.querySelector("[slot='confirm-button']") as HTMLElement;
```
with a parameterized "slot" name?

Reply:
Added a simple confirm dialog testing API